k8s/plugin/README.md

```diff
@@ -107,29 +107,41 @@ Options:
  pool-1-kworker1  aio:///dev/vdb?uuid=d8a36b4b-0435-4fee-bf76-f2aef980b833  true     kworker1  Online  500GiB    100GiB     400GiB
  ```
 
-7. Select pools based on labels. Filer labels must be provided in `key1=value1,key2=value2` format.
+7. Pool Labeling
 ```
-❯ kubectl mayastor get pools --selector  type=hhd
+❯ kubectl mayastor label pool pool-1-kworker1 zone-us=east-1
+Pool pool pool-1-kworker1 labelled successfully. Current labels: {"openebs.io/created-by": "operator-diskpool", "zone-us": "east-1"}
```

Comment:
hmm I wonder if we should hide the created-by label?

Reply:
sure, as its internal , we should not display it to the user